### PR TITLE
Improve wiki pages

### PR DESCRIPTION
**Pull request by intergrav**
Just some small changes to improve the grammar, that's all. Lmk what you think 👍🏽

### CODE REVIEW

**Comment by eristhea** [comment]
why does this remove so many newlines?
it makes the raw markdown a lot harder to read imo

**Comment by intergrav** [comment]
honestly idk, i found it easier to read without all the newlines but i could revert those if you want

**Comment by eristhea** [comment]
it's just my opinion but that would be nice!

**Comment by intergrav** [comment]
should be good now i think

File: wiki/amethyst_lantern.md [hunk 1]
```diff
@@ -3,10 +3,10 @@
 <!--description:Learn everything about the Amethyst Lantern, a very cute and practical lantern.-->
 <!--thumbnail:images/amethyst_lantern.png-->
 
-Amethyst Lantern is a type of lantern which primary color is purple, like amethyst!  
+Amethyst Lantern is a type of lantern which primary color is purple, like amethyst!
```
Comment (eristhea) [inline wiki/amethyst_lantern.md]:
```suggestion
The Amethyst Lantern is a type of lantern which primary color is purple, like amethyst! 
```

File: wiki/amethyst_lantern.md [hunk 2]
```diff
@@ -3,10 +3,10 @@
 <!--description:Learn everything about the Amethyst Lantern, a very cute and practical lantern.-->
 <!--thumbnail:images/amethyst_lantern.png-->
 
-Amethyst Lantern is a type of lantern which primary color is purple, like amethyst!  
+Amethyst Lantern is a type of lantern which primary color is purple, like amethyst! 
 They emit some purple particles while placed.
 
-In addition to the cosmetic part of this lantern, the Amethyst Lantern will cancel natural mob spawning in a 32 blocks radius in all directions around it.
+In addition to the cosmetic part of this lantern, the Amethyst Lantern will cancel natural mob spawning in a 32 blocks radius in all directions around it.
```
Comment (eristhea) [inline wiki/amethyst_lantern.md]:
```suggestion
In addition to its cosmetic function, the Amethyst Lantern will cancel natural mob spawning in a 32 block radius in all directions around it. 
```

File: wiki/blackboards.md [hunk 3]
```diff
@@ -4,12 +4,12 @@
...
 
-Blackboards are blocks which you can draw on with dyes.
+Blackboards are blocks which you can draw on with dyes. 
 They come in three variants: blackboard, chalkboard (green), and glassboard (transparent).
 
-They also can be worn as masks on the helmet slot, and on a dedicated slot if [Trinkets] is present.
+They can be placed, or worn as masks on the helmet slot, and on a dedicated slot if [Trinkets] is present.
 
-To place them they will need a support block, like candles, and they won't break if the support block is removed afterwards.
+To place them, they will need a support block like candles, and they won't break if the support block is removed afterwards.
```
Comment (eristhea) [inline wiki/blackboards.md]:
```suggestion
To be placed, they will need a support block like candles, and they won't break if the support block is removed afterwards.
```

File: wiki/painter_palette.md [hunk 4]
```diff
@@ -39,7 +39,7 @@ This item have an inventory capable of holding different blackboard-based <a hre
 The Painter's Palette contains an inventory capable of holding 4 drawing tools, and 27 [dye (including modifiers and paper)][dyes] items.
 The inventory can only hold one item per slot.
```
Comment (eristhea) [inline wiki/painter_palette.md]:
```suggestion
Its inventory can only hold one item per slot.
```

**Review by eristhea — APPROVED** [review]
just a few suggestions since you're already revising the pages

**Pull request merged**